### ISSUE TITLE
Relation widget starting 2.10.0-beta.5 up to current 2.10.1 no longer functions as expected

### ISSUE THREAD

**Issue by polds** [netlify/netlify-cms#2986]
**Describe the bug**

Starting at 2.10.0-beta.5 and present in 2.10.1 we've noticed that the relation widget no longer populates or completes results. I can see the XHR requests firing for lookups but all we see is the loading animation. 2.10.0-beta.4 and all prior betas and 2.9.x don't exhibit this. Additionally I don't see any evident javascript errors being written to the console.

We only have ~8 relations, so doesn't seem related to #2961.

**To Reproduce**

We're using (for verification) the unpkg versions:

 - https://unpkg.com/netlify-cms@2.10.0-beta.4/dist/netlify-cms.js <-- Working version
 - https://unpkg.com/netlify-cms@2.10.0-beta.5/dist/netlify-cms.js <-- Start break
 - https://unpkg.com/netlify-cms@2.10.1/dist/netlify-cms.js <-- Current version, still present

This is how we've defined our relation widget:

```yaml
collections:
  - name: "services"
    slug: '{{slug}}'
    summary: '{{Title}}'
    preview_path: "services/{{slug}}"
    label: "Our Services"
    label_singular: "Our Service"
    folder: "content/services"
    format: "toml-frontmatter"
    delete: false
    create: true
    identifier_field: Title
    fields:
      - label: "Title"
        name: "Title"
        widget: "string"
        required: true

  - name: "profiles"
    label: "People"
    label_singular: "Person"
    slug: '{{slug}}'
    summary: '{{Title}}'
    preview_path: "profiles/{{slug}}"
    folder: "content/profiles"
    format: "toml-frontmatter"
    create: true
    delete: true
    identifier_field: Title
    editor:
      preview: false
    fields:
      - label: "Services"
        name: "services"
        widget: "relation"
        collection: "services"
        searchFields: "Title"
        displayFields: "Title"
        valueField: "Service"
        required: true
        multiple: true
```

**Expected behavior**
It's expected when the cursor enters the relation widget a drop down of available relations are shown and if you type that an auto complete will happen.

**Screenshots**
**2.10.0-beta.4:**
![](http://dev.imgt.io/eJC/RpVsw1.png)

**2.10.0-beta.5 (and beyond):**
![](http://dev.imgt.io/xkD/RpVs1.png)

**Applicable Versions:**
 - Netlify CMS version: >= 2.10.0-beta.5
 - Git Provider: Gitlab
 - OS: Verified in MacOS
 - Browsers: Chrome 78, Safari

**Comment by erquhart** [comment]
Fix wasn’t released correctly, will be out soon.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by erquhart** [comment]
Fixed in netlify-cms@2.10.2-beta.0 and netlify-cms@2.10.2.
